find_path.py

```diff
@@ -10,6 +10,9 @@
 #Go from label to entity_uri (for PKL original labels file) or Label to Idenifier (for microbiome PKL)
 def get_uri(labels,value):
 
+    print(value)
+    print(labels.columns)
```

Comment:
Is this just for testing?

Reply:
Thanks for catching that, yes it is now removed.